nextflow.config

```diff
@@ -8,352 +8,654 @@
 // Global default params, used in configs
 params {
   // Workflow flags
+  publishDir_intermediate=false // set to true to publish all the middle stages
+  perform_identification=false // Set to false if you don't want to do identification. You will not require to set MS2 related parameters if you set this to false
+  perform_identification_metfrag=false // Should Metfrag be used for doing identification?
+  perform_identification_csifingerid=false // Should CSI:FingerID be used for doing identification?
+  perform_identification_cfmid=false // Should CFM-ID be used for doing identification?
+  perform_identification_internal_library=false // Should an internal library be used for doing identification?
+
+
+  type_of_ionization="neg" // You can either set to 'pos' (only positive), 'neg' (only negative), 'both' (both positive and negative).
+  need_centroiding=false // Set to true if your data is in profile mode (only for quantification!)
+
+  quantification_openms_xcms_pos="xcms" // set whether you want to do quantification with OpenMS (openms) or XCMS (xcms) in positive ionization
+  quantification_openms_xcms_neg="xcms" // set whether you want to do quantification with OpenMS (openms) or XCMS (xcms) for negative ionization
+
+
+  peakpicker_ini_file_pos_openms="$baseDir/assets/openms/openms_peak_picker_ini_pos.ini" //Path to the ini file for PeakPickerHiRes
+  peakpicker_ini_file_neg_openms="$baseDir/assets/openms/openms_peak_picker_ini_neg.ini" //Path to the ini file for PeakPickerHiRes
+
+  featurefinder_ini_pos_openms="$baseDir/assets/openms/openms_feature_finder_metabo_ini_pos.ini" // Path to the ini file for OpenMS FeatureFinderMetabo in positive mode
+  featurefinder_ini_neg_openms="$baseDir/assets/openms/openms_feature_finder_metabo_ini_neg.ini" // Path to the ini file for OpenMS FeatureFinderMetabo in negative mode
+
+  quantification_openms_xcms_library_pos="xcms" // set whether you want to do quantification with OpenMS (openms) or XCMS (xcms) in positive ionization  (for library)
+  quantification_openms_xcms_library_neg="xcms" // set whether you want to do quantification with OpenMS (openms) or XCMS (xcms) for library (negative)
```

Comment:
These appear to be duplicated again - are they not already in `conf/parameters.conf` already? Either import that file with `importConfig` and remove their definition from here, or delete that file and keep them just in this file.

These parameters should only be defined with a default once in one place, any other reference should only be to overwrite them in given config profiles.